config/environments/development.rb

```diff
@@ -33,7 +33,7 @@
   # Store uploaded files on the local file system (see config/storage.yml for options).
   config.active_storage.service = :local
 
-  config.action_mailer.perform_deliveries = true
+  config.action_mailer.perform_deliveries = false
   config.action_mailer.raise_delivery_errors = true
   config.action_mailer.delivery_method = :smtp
```

Comment:
```suggestion
  config.action_mailer.perform_deliveries = true
  config.action_mailer.raise_delivery_errors = true
  config.action_mailer.delivery_method = :letter_opener
```

I recommend using the gem `letter_opener` as this will allow you to previous the emails in your localhost ;-)